validate.php

```diff
@@ -219,6 +219,8 @@
     $cmd = rtrim(shell_exec("which {$config[$bin]} 2>/dev/null"));
     if (!$cmd) {
         print_fail("$bin location is incorrect or bin not installed");
+        print_fail("You can also manually set the path to $bin by placing the following in config.php:");
+        print_fail("\$config['$bin'] = \"/path/to/$bin\";");
```

Comment:
See how it's done here https://github.com/andrewwippler/librenms/blob/4d3120b74a682de7748bc06188e19e85aa79e4f5/validate.php#L147